src/components/Project/ProjectDetailCard.js

```diff
@@ -0,0 +1,523 @@
+import React from 'react';
+
+export default function ProjectDetailCard() {
+  return (
+    <div className='main-panel'>
+      <div className='content'>
+        <div className='container-fluid'>
+          <div className='row'>
+            <div className='col-md-10'>
+              <div className='card'>
+                <div className='card-header card-header-warning'>
+                  <h4 className='card-title'>Lesewert 2018</h4>
+                  <p className='category'>
+                    Project ID: <span>2018-33</span>
```

Comment:
Need to consume API and use React, as I can see now it's fully static

Reply:
Postponed https://trello.com/c/BgfSPOet

Reply:
https://trello.com/c/fpnoYKTU